README.md

```diff
@@ -59,16 +61,21 @@ On Windows, use `.\py3\Scripts\activate.bat` instead of `source ./py3/bin/activa
       -m _MAP_USERS, --map-user _MAP_USERS
                             Override user mapping for usernames, for example
                             `--map-user fk=fkrull`. Can be specified multiple
-                            times.
+                            times. Disable mapping for a user with an empty value,
+                            for example `--map-user fk=`.
 
       --skip-attribution-for BB_SKIP
-                            BitBucket user who doesn't need comments re-
-                            attributed. Useful to skip your own comments, because
-                            you are running this script, and the GitHub comments
-                            will be already under your name.
+                            Your BitBucket username, if you don't want comments
```

Comment:
Hmm... been a bit since I've looked at this code, but I thought this isn't just for _your_ bb username... it can be applied to any username?

Also, your updated comment for `--map-user` suggests that this flag may not even be needed? Since you can just map your own name to an empty value? Is that correct?

Reply:
The code as it stands only really works for _your_ username, particularly since the [issue template attributes the issue to "me"](https://github.com/jeffwidman/bitbucket-issue-migration/blob/d83a92b/migrate.py#L511). Even if the template was changed, though, it doesn't really make sense to pass any other user, since it can only be set once and without attribution the comment effectively is being attributed to the user running the script.